src/main/java/com/polymars/game/Cards.java

```diff
@@ -0,0 +1,13 @@
+package com.polymars.game;
+
+public class Cards {
+
+    public static final int FIRE = 0;
+    public static final int WATER = 1;
+    public static final int SNOW = 2;
+    public static final int MIXED = 3;
+
+    public static final int ORANGE = 1;
+    public static final int PURPLE = 5;
+
+}
```

Comment:
Why not make this an enum? You claim to optimize everything and this seems like easy performance